endoreg_db/models/requirement/requirement_evaluation/__init__.py

```
@@ -0,0 +1,75 @@
+    """
+    Extracts values from the provided keyword arguments based on the requirement.
+
+    Args:
+        requirement (Requirement): The requirement to evaluate.
+        **kwargs: Additional keyword arguments for evaluation.
+
+    Returns:
+        dict: A dictionary containing the extracted values.
+    """
+    requirement_types = [_.name for _ in requirement.requirement_types]
+    operators = [_.name for _ in requirement.operators]
```

Ruff (0.8.2)

Local variable `operators` is assigned to but never used

Remove assignment to unused variable `operators`

(F841)